### ISSUE TITLE
Firefox addon is out of date

### ISSUE THREAD

**Issue by chromer030** [inbasic/media-player#37]
Thank you for this great and useful extension , Firefox version is stuck on version 0.2.0 and Last updated is a year ago (May 22, 2021) , please update it to latest version like chrome.

Thank you very much.

https://addons.mozilla.org/en-US/firefox/addon/the-media-player/

**Comment by inbasic** [comment]
The 0.4.3 release was rejected by AMO due to the use of the "cast" plugin. I'll submit a new version without this plugin, and hopefully, it gets passed.

**Comment by chromer030** [comment]
Thank you very much for your efforts , New version without some features is far better than old version without some features.

**Comment by chromer030** [comment]
@inbasic , any update ?

**Comment by inbasic** [comment]
I tried several times to push the latest release. Each time I get a rejection letter like the following. Although there are references where the libs are taken from (see README files next to the libs), the reviewer wants the official link. The problem is that these libraries do not provide official links. They recommended using the CDN version, but the reviewer does not accept them. Have no idea how to proceed.

```
Details:
Established libraries must be included from their official source, in their original format without any modification (changing the file name does not matter). Please note that only stable releases are acceptable (not beta, pre, RC, dev etc) and that third party CDNs are not considered official sources.
- data\player\video.js\plugins\wavesurfer\wavesurfer.js
- data/player/video.js/plugins/playlist/videojs-playlist.js
- data/player/video.js/plugins/playlist-ui/videojs-playlist-ui.js

Version(s) affected and disabled:
0.4.4.1
********
```

**Comment by chromer030** [comment]
So please provide .xpi file Github release for every version , so users can download and install manually.
With .xpi files every feature will be available like cast !

Thanks.

**Comment by chromer030** [comment]
It's possible to drag and drop and install firefox .xpi addons without any issues :
![Screenshot_20220822_150042](https://user-images.githubusercontent.com/13361552/185900738-e7462a7a-3eee-4734-9801-488f561d1ffc.jpg)

**Comment by inbasic** [comment]
It is not possible! XPI needs to be signed. Without the signing, you cannot drag and drop to install.

**Comment by chromer030** [comment]
You are right , i didn't notice that , even prominent Ublock Origin XPI files released in Github are signed.

**Comment by NielsNL12** [comment]
@inbasic Have you tried to resubmit since then? I'd like to try the add-on, but it's very out-of-date.
